### ISSUE TITLE
Update links for explanations on Transfers and Adressess page

### ISSUE THREAD

**Issue by nop33** [alephium/desktop-wallet#644]
Currently, those links lead to our FAQ. I guess we need to write some entries regarding those pages? @mvaivre what was the rational behind this feature?

(top right blue boxes)

<img width="1608" alt="image" src="https://user-images.githubusercontent.com/1579899/234605767-a1dd529f-50df-4bb6-a8ab-dd38ae473b73.png">
<img width="1639" alt="image" src="https://user-images.githubusercontent.com/1579899/234605805-ab9270b8-1689-4da9-b5e8-ce8578929013.png">

**Comment by mvaivre** [comment]
Yes it's a placeholder to redirect to the documentation articles that... don't exist yet. 
We can hide the boxes until the docs are updated.

Let's keep this issue to remind us to reactivate the feature once we have corresponding articles.

**Comment by nop33** [comment]
Do we wanna transfer this issue to the monorepo @mvaivre ?
